### DISCUSSION TITLE
How to query raw BSON type in MongoDB like this "Timestamp(1412180887, 3)"

### DISCUSSION THREAD

**Discussion by un4ckn0wl3z** [prisma/prisma#19126] [Q&A] [answered]
I need raw BSON from MongoDB (unix timestamp and ordinal number). Is Prisma support this?

**Comment by ludralph** [comment, marked as answer]
Hi @un4ckn0wl3z 👋 

Prisma maps BSON types to Prisma Scalar types. So I don't think we support raw BSON from MongoDB. However, you can open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) for this.

**Reply by un4ckn0wl3z** [reply]
Ok sir, Thanks for response /\
